doc/man3/RAND_add.pod

```diff
@@ -78,6 +69,54 @@ RAND_event() returns RAND_status().
 
 The other functions do not return values.
 
+=head1 NOTES
+
+The new OpenSSL DRBG has some peculiarities which need to be taken
+into account when it is selected as the default OpenSSL CSPRNG, i.e.,
+when RAND_get_rand_method(void) == RAND_OpenSSL(). This applies in
```

Comment:
I recommend breaking lines at the end of sentences, so between the period and "This" here. That's to get around the debate about the amount of space between sentences.